### PR TITLE
Migrate to Yarn 3

### PR DESCRIPTION
**Pull request by denysdovhan**
## Summary

* Migrate to Yarn 3 (#9)
* Drop Lerna in favor of workspace tools

@Fer0x could you verify node binaries will be linked properly. You said there should be some option set in configs in order to do that?

## How did you test this change?

Just use Yarn 🤷🏻

### CODE REVIEW

**Comment by denysdovhan** [comment]
@Fer0x seems like the problem with resolving dev deps in child packages is present here. That's why CI is failing at the moment.

File: package.json [hunk 1]
```diff
@@ -7,11 +7,12 @@
...
     "commit": "cz",
     "prepare": "husky install",
-    "build": "lerna run build --stream --parallel",
-    "test": "lerna run test --stream --parallel",
-    "clear": "lerna run clear --stream --parallel",
-    "release": "lerna run release --stream"
+    "build": "yarn workspaces foreach -pti --exclude @autoviews/docs run build",
+    "clear": "yarn workspaces foreach -pi run clear",
+    "release": "yarn workspaces foreach -i run release",
+    "test": "is-ci && jest --maxWorkers=7 || jest --maxWorkers=50%"
```
Comment (Fer0x) [inline package.json:13]: `--maxWorkers=7` is requirement for internal Wix CI. Also, don't know if  any `--maxWorkers` affects jest performance on GitHub Actions

File: .gitignore [hunk 2]
```diff
@@ -15,7 +15,13 @@ build
...
 .docusaurus
 .cache-loader
-.yarn
+
+# Yarn
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.yarn/install-state.gz
+.pnp.js
```
Comment (Fer0x) [inline .gitignore:24]: better use ignores from this guide https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

**Review by Fer0x — CHANGES_REQUESTED** [review]

**Comment by denysdovhan** [comment]
Closing this, because yarn 3 is not ready to be implemented.

**Pull request closed without merging**